src/components/photoIndexIndicator/PhotoIndexIndicator.tsx

```diff
@@ -0,0 +1,24 @@
+import { PhotoIndexIndicatorAbove5, renderPhotoIndexIndicator } from './helper';
+import { st, classes } from './PhotoIndexIndicator.st.css';
+
+interface IPhotoIndexIndicator {
+    currentImgUrlIndex: number;
+    imgUrls: string[];
+}
+const PhotoIndexIndicator = ({ currentImgUrlIndex, imgUrls }: IPhotoIndexIndicator) => {
+    const normalDot = <div className={classes.dot}>&#8226;</div>;
+    const currentDot = <div className={st(classes.dot, classes.currentDot)}>&#8226;</div>;
+    const fadeDot = <div className={st(classes.dot, classes.fadeDot)}>&#8226;</div>;
+
+    return imgUrls.length < 6 ? (
```

Comment:
Why is this 6? could be 7.
even 5 if your feeling brave.

have this be an optional parameter for this component which defaults to 6.
theres no reason this should be infinitely scaleable

Reply:
only for display. 
It scales, it will work with millions images, but its behavior different => if it length is bigger then 5.

Reply:
move 6 to props from father